nuget/SDammann.WebApi.Versioning.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>SDammann.WebApi.Versioning</id>
 	<title>ASP.NET Web API versioning support</title>
-    <version>2.7.0.0</version>
+    <version>2.9.1.1</version>
```

Comment:
Since 2.7 is the current version I won't merge this and just keep the existing increment